src/commits.js

```diff
@@ -54,7 +57,7 @@ function parseCommit (commit, remote, options = {}) {
     shorthash: hash.slice(0, 7),
     author,
     email,
-    date,
+    date: date.replace(MATCH_DATE, DATE_TO_STRICT_ISO_REPLACE_PATTERN),
```

Comment:
For backward compatibility, transforming dates from ISO 8601-like format to strict ISO 8601.